test/cql-pytest/test_in_operator.py

```diff
@@ -85,7 +86,8 @@ def test_in_query_with_valid_product_limit(cql: Session, rows: List[Any], table:
     assert len(cql.execute(query).current_rows) == len(rows)
 
 
-def test_in_query_with_invalid_product_limit(cql: Session, scylla_only, table: str):
+@scylla_only
```

Comment:
Ok, so after the long discussion on whether this is a valid test or not, we've come to this. There is nothing wrong here - indeed this test will pass on today's Scylla and be skipped (because it isn't expected to pass) on Cassandra. HOWEVER, if we ever change the default arbitrary limit from 100 to anything else, this test will begin to fail. If we ever fix the weird error message we print in this case ("NoHostAvailable"?!), this test will fail. 
On one hand, I'm not sure it's a great idea to have a test which enshrines broken or arbitrary behavior, but on the other hand - that's the whole idea of keeping tests inside scylla.git - if we ever change this behavior, we can easily change the test in the same patch. So it's ok to have such tests.